lib/plugs/interval/time_to.ex

```diff
@@ -1,16 +1,13 @@
 defmodule Castle.Plugs.Interval.TimeTo do
 
-  # if no end date is set, just use this many seconds into the future
-  @buffer_seconds 60
-
   def parse(%{params: %{"to" => to_dtim}}) when is_bitstring(to_dtim) do
     case parse_dtim(to_dtim) do
       {:ok, dtim} -> {:ok, Timex.Timezone.convert(dtim, :utc)}
       {:error, _err} -> {:error, "Bad to param: must be a valid ISO8601 date"}
     end
   end
   def parse(_conn) do
-    {:ok, Timex.shift(Timex.now, seconds: @buffer_seconds)}
+    {:ok, Timex.now |> Timex.beginning_of_day |> Timex.shift(days: 1)}
```

Comment:
Metrics always sets a `?to=` date.  But I found that if you didn't, Castle would end up picking one rounded to the "next hour from now".  Which would cause the `?interval=DAY` to pull from the hourly cache, rather than the daily cache (see `plugs/interval/rollups.ex:17`).

This will return the same results, but always use the daily cache (which is faster).